src/main/resources/assets/identity/lang/zh_cn.json

```
@@ -1,13 +1,25 @@
 {
-  "identity.unlock_entity": "已解锁%s种化身",
-  "identity.revoke_entity": "你的对%s化身的记忆已化为尘埃...",
-  "identity.grant_success": "成功将化身%s给予%s。",
-  "identity.revoke_success": "成功删除%s化身(%s身上的化身)。",
-  "identity.already_has": "%s已经拥有%s化身！",
-  "identity.does_not_have": "%s没有%s化身！",
-  "identity.equip_success": "将%s的化身设置为%s。",
-  "identity.unequip_success": "已删除%s化身。",
-  "identity.test_positive": "%s化身为%s。",
-  "identity.test_failed": "%s没有化身为%s",
-  "identity.menu_hint": "杀死任何生物以获得化身..."
-}
+    "identity.unlock_entity": "已解锁%s的化身",
+    "identity.revoke_entity": "你化身为%s的记忆已逐渐消逝......",
+    "identity.grant_success": "已成功将%s的化身转给%s。",
+    "identity.revoke_success": "已成功将%s的化身转回给%s。",
+    "identity.already_has": "%s已经拥有%s的化身了！",
+    "identity.does_not_have": "%s还没有%s的化身！",
+    "identity.equip_success": "%s的化身已装备给%s。",
+    "identity.unequip_success": "已移除%s的化身。",
+    "identity.test_positive": "%s已经伪装成%s了。",
+    "identity.test_failed": "%s还没被伪装成%s。",
+    "identity.menu_hint": "杀死任意一种生物来在你的世界里获得化身。",
+    "identity.help": "打开《化身》模组帮助按钮",
+    "identity.help.welcome": "欢迎使用《化身》，这是模组《变形（Morph）》的1.16+ Fabric版本，由Draylar编写。",
+    "identity.help.credits": "感谢所有通过提交bug和建议来支持我的模组的人们！",
+    "identity.help.support_label": "遇到问题还是想提供建议？",
+    "identity.help.support_description": "请到《化身》的Github存储库来提交 https://github.com/Draylar/identity 。",
```

```suggestion
    "identity.help.support_description": "通过《化身》的 GitHub 仓库 https://github.com/Draylar/identity 交流。",
```
- GitHub, not Github
- 印象里“仓库”的说法比“存储库”的说法用的多一些？
- "reach out to" 有（为了帮助）与（某人）沟通的意思，译为“提交”不妥